zul/src/archive/web/js/zul/mesh/MeshWidget.js

```diff
@@ -800,6 +800,7 @@ zul.mesh.MeshWidget = zk.$extends(zul.Widget, {
 			if (this._cssflex && this.isChildrenFlex()) return;
 			var empty = true,
 				flex = false,
+				shouldFix = false,

```

Comment:
shouldCheckEmptyWidth = !this.frozen && this.eheadtbl && this.eheadtbl.style.width